tasks/main.yml

```diff
@@ -5,33 +5,78 @@
 
 - include: "install-{{ ansible_os_family }}.yml"
 
-- name: Create log directory
-  file:
-    path: "{{ dpb_log_dir }}"
-    mode: 0755
-    owner: "{{ dpb_user }}"
-    group: "{{ dpb_group }}"
-    state: directory
-  notify: Restart dpb
+# Unset "nodev" mount(8) option if /usr/local is mounted with the option before
+# creating chroot because, inside of chroot, device nodes must be created.
+- name: Get mount option of /usr/local
+  # translation:
+  #
+  # * get `mount(8)` output
+  # * remove unnecessary lines starting with '#'
+  # * select an entry with mount point == "/usr/local"
+  # * remove everything other than options
+  #
+  # the result is "opt1,opt2"
+  shell: "mount | grep -v '^#' | awk '{if ($3 == \"/usr/local\") { print $0 }}' | sed -e 's/^.*(//' -e 's/)$//' -e 's/[ ]//g'"
```

Comment:
* when does mount(8) print lines starting with '#'?
* awk or sed can do everything. is there any reason to use grep, sed and awk?
* multiple commands can be executed with a `-e` like `-e 's/.../.../;s/.../.../'`
* or, `sed -e 's/.*(\(.*\)).*/\1/' `.